test/integration/activate_int_test.go

```diff
@@ -51,6 +51,23 @@ func (suite *ActivateIntegrationTestSuite) TestActivateWithoutRuntime() {
 	cp.ExpectExitCode(123, 10*time.Second)
 }
 
+func (suite *ActivateIntegrationTestSuite) TestActivateNotOnPath() {
+	ts := e2e.NewNoPathUpdate(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.Spawn("activate", "ActiveState-CLI/Python3")
+	cp.Expect("Where would you like to checkout")
+	cp.SendLine(cp.WorkDirectory())
+	cp.Expect("activated state", 20*time.Second)
+	cp.WaitForInput(10 * time.Second)
+
+	cp.SendLine("alias state")
```

Comment:
This won't pass on Windows. Please run integration tests on this PR (add the label).

Reply:
Sorry, I was still working out the doskey args. It helped me fix (hopefully) a bug where args were not passed to the aliased command.